### DISCUSSION TITLE
Scaling Text

### DISCUSSION THREAD

**Discussion by hilltopmediadesign** [uxbox/uxbox#821] [Q&A] [answered]
First off, amazing work!

Is there a way to scale text? As in keyboard command 'k' in figma.

Thanks.

**Comment by myfunnyandy** [comment]
Not at the moment, but it makes a lot of sense so we have added to our backlog: https://tree.taiga.io/project/penpot/us/1527?no-milestone=1

**Comment by scottonanski** [comment]
Is this implemented yet?

**Reply by madalenapmelo-kp** [reply, marked as answer]
Yes, this has already been implemented!

**Reply by myfunnyandy** [reply]
It works for texts and also effects and other properties (blur, shadow, stroke...) https://help.penpot.app/user-guide/layer-basics/#scale-elements
